packages/sdk-middleware-auth/test/build-requests.spec.js

```diff
@@ -31,6 +31,28 @@ describe('buildRequestForClientCredentialsFlow', () => {
     })
   })
 
+  it('parses a host that ends with \'/\'', () => {
```

Comment:
@emmenko The eslint errors are because I replaced ```'``` with ``` ` ``` to avoid escaping and make the code more reasonable (I forced the commit)
Would replace ```/``` with ```"slash"``` to maintain readability and lintability... Then everyone should be happy 😋